lib/spells/effects/Catapult.cpp

```diff
@@ -113,7 +112,7 @@ void Catapult::apply(ServerCallback * server, const Mechanics * m, const EffectT
 
 		if (attackInfo == ca.attackedParts.end()) // new part
 		{
-			CatapultAttack::AttackInfo newInfo;
+			CatapultAttack::AttackInfo newInfo; //NOLINT - braced init will not work due to -Wnarrowing
```

Comment:
Not a big fan of having comments that are not comments but metainfo.

Reply:
@kambala-decapitator this comment will block clang-tidy from adding a bracing empty init.

I tried to add a braced init myself - but there is -Wnarrowing, because battleHex will return 16-bit, but structure accepts 8 bit.